js_pt1/dest/desktop/js/script5+1.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+	'use strict';
+	kintone.events.on('app.record.create.show', function(event) {
+		let actions = [
+			'Relentless quest',
+			'Mental and physical persistence',
+			'Empathy for the ideal',
+			'Inspire others',
+			'Increase knowledge and skills',
+			'Fairness and honesty'
+		]
+		let rows = [];
+		for (let i = 0; i < 6; i++) {
+			rows.push(createRow(actions[i]));
+		}
+		event.record.table.value = rows;
+		return event;
+	});
+
+	function createRow(action) {
+		return {
+			'value': {
+				'action_5': {
+					'type': 'DROP_DOWN',
+					'value': action,
+				},
+				'status': {
+					'type': 'CHECK_BOX',
+					'value': ['Not reviewed']
+				},
+				'task': {
+					'type': 'MULTI_LINE_TEXT',
+					'value': undefined
```

Comment:
The value of the `'value'` should be `''` as the value of the MULTI_LINE_TEXT field is `''` when it's empty.
Please check the value of MULTI_LINE_TEXT field when it's empty.